beta-src/src/components/map/WDMapController.tsx

```diff
@@ -39,8 +39,10 @@ const WDMapController: React.FC<WDMapControllerProps> = function ({
     if (svgElement.current) {
       const fullMap = d3.select(svgElement.current);
       const contained = fullMap.select("#container");
-      const containedRect = contained.node().getBBox();
-      const gameBoardAreaRect = fullMap.select("#outlines").node().getBBox();
+      const containedRect = (contained.node() as SVGSVGElement).getBBox();
+      const gameBoardAreaRect = (
+        fullMap.select("#outlines").node() as SVGSVGElement
+      ).getBBox();
```

Comment:
Isn't this pointless unless you're using @types/d3?

Reply:
I was going to remove this when cleaning up, but figured it would be nice to force the SVG type for that node as it needs to be an svg for BBox if I am not mistaken